server_admin/topics/users/user-registration.adoc

```diff
@@ -1,15 +1,16 @@
-[id="con-user-registration_{context}"]
+[id="assembly-user-registration_{context}"]
+= User Registration
 
-=== User Registration
+[role="_abstract"]
+You can use {project_name} as a third-party authorization server to manage application users. Administrators can add users or users can register themselves.
```

Comment:
This "third-party" sentence sounds good, but I am not sure where this came from.

Reply:
I got that from the Keycloak docs - keycloak.org and baeldung.com

Reply:
Excellent. Thanks for researching it.